parm/config/config.waveprep

```diff
@@ -20,20 +20,24 @@ export POFILETYPE=1     # POFILETYPE=1 one point file per output step
 # Unified output T or F
 export FUNIPNT='T'
 # Output server type (see ww3_multi.inp in WW3 repo)
-export IOSRV='3'
+export IOSRV='1'
```

Comment:
Why is this being changed?

Reply:
Example config.waveprep has it set to 1, see: /work/noaa/global/wkolczyn/save/global-workflow/expdir/example/config.waveprep
If it should remain set to 3 let me know, thanks!

Reply:
Since we were keeping everything as this is the develop/gfsv16 case and not updating to use the "coupled"/s2s case, this should be reverted to what's in develop.

Reply:
Changing back to 3, thanks!